cmd/security-file-token-provider/res/token-config.json

```diff
@@ -175,5 +175,8 @@
         }
       }
     }
+  },
+  "rules-engine": {
+    "edgex_use_defaults": true
```

Comment:
Why is this needed? 
should be `app-rules-engine` if really needed

Reply:
my understanding is that app-rules-engine and ekuiper are not the same, right? I used the name of the service as found in the docker-file which is `rules-engine:` for ekuiper. it also follows along with the REST requests which are performed by the UI which issues a GET to `http://localhost:4000/rules-engine/ping`